index.html

```diff
@@ -48,8 +50,252 @@ <h1 class="main-name">Erik Stoupignan</h1>
+                        </div>
+                        <!-- Button's Code -->
+                        <div class="box align box3">
+                            <p class="code-button">HTML/CSS</p>
+                        </div>
+                        <div class="box align box4">
+                            <p class="code-button">Ruby on Rails</p>
+                        </div>
+                        <div class="box align box5">
+                            <p class="code-button">JavaScript</p>
+                        </div>
+                        <!-- See this project button -->
+                        <div class="box align-button box6">
+                            <a href="#">
+                                <div class="buttom-arrow">
+                                    <p>See this project
+                                    <img class="arrow" src="/assets/211621_c_right_arrow_icon.png" alt="Right arrow">
+                                    </p>
+                                </div>
+                            </a>
+                        </div>
+                    </div>
+                </li>
+            </ul>
+        </div>
+    </section>
+
+    <!-- About me -->
+    <section class="about">
+        <h1>About</h1>
+        <p class="main-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit
+            , sed do eiusmod tempor incididunt ut labore et dolore
+            magna aliqua. Ut enim ad minim veniam, quis nostrud 
+            exercitation ullamco laboris nisi ut aliquip ea 
+            eiusmod
+        </p>
+    </section>
+
+    <!-- Gey my CV -->
+    <div class="">
+        <div class="start-collaboration">
+            <a class="cv_link" href="https://github.com/ErikStoupignan">
+                <p>Get my CV
+                <img class="arrow" src="/assets/downloadicon.png" alt="Download Icon">
+                </p>
+            </a>
+        </div>
+    </div>
+
+    <!-- Code lenguajes -->
+    <div class="skills">
+        <div class="square leng-color">
+            <p class="skill-font-title">Languages</p>
+            <ul class="skill-font-ul">
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+        </div>
+
+        <div class="square fram-color">
+            <p class="skill-font-title">Frameworks</p>
+            <ul class="skill-font-ul">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Selenium</li>
+            </ul>
+        </div>
+
+        <div class="square skill-color">
+            <p class="skill-font-title">Skills</p>
+            <ul class="skill-font-ul">
+                <li>Codekit</li>
+                <li>GitHub</li>
+                <li>GitLAb</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+    </div>
```

Comment:
This whole section is the about-me section, kindly wrap this whole section in a `section` tag.

Reply:
Thankk you @muneebulrehman.
You are absolutely right, sorry. I didn't think things clear.